benchbuild/environments/service_layer/ui.py

```diff
@@ -0,0 +1,60 @@
+import typing as tp
+
+from rich.progress import Task, Progress, BarColumn
+
+from benchbuild.environments.domain import events
+from benchbuild.environments.service_layer import unit_of_work
+
+
+def progress_print_image_created(
+    event: events.ImageCreated, uow: unit_of_work.AbstractUnitOfWork
+) -> None:
+    del uow
+    __progress__.console.print(f'Building {event.name}')
+    # __tasks__[event.name
+    #          ] = __progress__.add_task(event.name, total=event.num_layers - 1)
+
+
+def progress_print_creating_layer(
+    event: events.CreatingLayer, uow: unit_of_work.AbstractUnitOfWork
+) -> None:
+    del uow
+    __progress__.console.print(event.layer)
```

Comment:
This seems a bit confusing to me, what is the idea behin calling `progress_print..` and the calling `del`.
What is the idea behind this?

Reply:
This is the pattern used to signal an unused argument without disabling the linter warnings.

Reply:
fixed.